.changeset/spicy-bees-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/cryptocompare-adapter': patch
```

Comment:
Still need to review the rest of this PR, but can we make this a major update (as it may cause breaking changes if NOPs have their `WS_API_ENDPOINT` set)
```suggestion
'@chainlink/cryptocompare-adapter': major
```